python/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
```

Comment:
General question - what is our strategy on `poetry.lock`. 
pro - it pins the resolution of dependencies, and reduces change of varying resolution when installed by end user
cons - it can be affected by the last dev who updates (for example here the version used differs - minor), unless done in ci
...?

Reply:
I would pin it definitelly.